tests/App/config/config.yml

```diff
@@ -28,6 +28,7 @@ parameters:
 doctrine:
     dbal:
         url: "%env(resolve:DATABASE_URL)%"
+        use_savepoints: true
```

Comment:
Enabled for: https://github.com/dmaicher/doctrine-test-bundle/pull/249

They seem to be required for Doctrine Test Bundle but also enabled by default on DBAL 4 (deprecated not enabling them on DBAL 3)

Reply:
It was more related to doctrine test bundle 7.0 not supporting the use of savepoints (I think), because now the build works using only 8.0